orttraining/orttraining/training_ops/cuda/loss/softmax_cross_entropy_loss_impl.cu

```diff
@@ -221,6 +223,7 @@ SPECIALIZED_IMPL_SoftMaxEntropyLossGradImpl(float, int64_t)
 
 SPECIALIZED_IMPL_ComputeWeightsSoftmaxCrossEntropyImpl(float, int32_t)
 SPECIALIZED_IMPL_ComputeWeightsSoftmaxCrossEntropyImpl(float, int64_t)
+SPECIALIZED_IMPL_ComputeWeightsSoftmaxCrossEntropyImpl(half, int64_t)
```

Comment:
Historically, "specialized" was used here. Actually, it is to instantiate the template function. Can you help change to INSTANTIATE_XXX?